### PR TITLE
Update docker fixtures for Java 11

### PR DESCRIPTION
**Pull request by timja**
ATH is failing on recent core

see https://github.com/jenkinsci/acceptance-test-harness/issues/807

### CODE REVIEW

**Review by MarkEWaite — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]
Thanks a bunch for addressing, though looks like spotbugs got much more strict 👀

**Comment by timja** [comment]
There’s a test failing due to a locale change at least for me locally will look later

File: src/test/java/org/jenkinsci/test/acceptance/docker/fixtures/SshdContainerTest.java [hunk 1]
```diff
@@ -50,12 +51,10 @@ public void smokes() throws Exception {
     public void locale() throws Exception {
         JavaContainer c = javaContainerRule.get();
         // cf. https://stackoverflow.com/a/4384506/12916
-        assertThat(c.popen(new CommandBuilder("jrunscript", "-e", "'println(java.lang.System.getProperty(\"file.encoding\") + \" vs. \" + java.lang.System.getProperty(\"sun.jnu.encoding\"))'")).verifyOrDieWith("could not run jrunscript"),
```
Comment (timja) [inline src/test/java/org/jenkinsci/test/acceptance/docker/fixtures/SshdContainerTest.java:53]: jrunscript is deprecated for removal in 11 and removed in 14.

File: src/test/java/org/jenkinsci/test/acceptance/docker/fixtures/SshdContainerTest.java [hunk 2]
```diff
@@ -50,12 +51,10 @@ public void smokes() throws Exception {
...
+        assertThat(c.popen(new CommandBuilder("echo", "'System.out.println(System.getProperty(\"file.encoding\") + \" vs. \" + System.getProperty(\"sun.jnu.encoding\"))'", "| jshell -")).verifyOrDieWith("could not run jshell"),
             containsString("UTF-8 vs. UTF-8"));
-        assertThat(c.popen(new CommandBuilder("jrunscript", "-e", "'var f = new java.io.File(\"hello\\u010d\\u0950\"); new java.io.FileOutputStream(f).close(); println(\"name: \" + java.net.URLEncoder.encode(f.name, \"UTF-8\")); println(\"exists: \" + f.file)'")).verifyOrDieWith("could not run jrunscript"),
+        assertThat(c.popen(new CommandBuilder("echo", "'var f = new java.io.File(\"hello\\u010d\\u0950\"); new java.io.FileOutputStream(f).close(); System.out.println(\"name: \" + java.net.URLEncoder.encode(f.getName(), \"UTF-8\")); System.out.println(\"exists: \" + f.exists())'", "| jshell -")).verifyOrDieWith("could not run jshell"),
             allOf(containsString("exists: true"), containsString("name: hello%C4%8D%E0%A5%90")));
-        assertThat(c.popen(new CommandBuilder("sh", "-c", "'ls | native2ascii -encoding UTF-8'")).verifyOrDieWith("could not run ls"),
```
Comment (timja) [inline src/test/java/org/jenkinsci/test/acceptance/docker/fixtures/SshdContainerTest.java:57]: removed in 9, https://bugs.openjdk.org/browse/JDK-8074431

File: pom.xml [hunk 3]
```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci</groupId>
         <artifactId>jenkins</artifactId>
-        <version>1.54</version>
+        <version>1.76</version>
```
Comment (timja) [inline pom.xml:7]: the pom file wouldn't build on Java 11 without some updates.

I didn't bother going to latest which requires Java 11 as we still want to use this library on 8 for a bit for ATH.
I know I could add some trickery but I don't think needed for this repo.

**Comment by timja** [comment]
> There’s a test failing due to a locale change at least for me locally will look later

heh, that just fails in my IDE I didn't manage to fix it, but via maven it works fine and the only issue was native2ascii getting removed

**Comment by timja** [comment]
@jglick  / @Vlatombe any chance of a quick release please? this is breaking ATH since Java 11 move.

(very surprised this wasn't tested -.-)

File: pom.xml (hunk 3 above)
Comment (jglick) [reply pom.xml:7]: Yeah it is deprecated anyway.

**Review by jglick — APPROVED** [review]

**Comment by jglick** [comment]
https://github.com/jenkinsci/docker-fixtures/releases/tag/docker-fixtures-1.12

**Pull request merged**